src/argprocessing.cpp

```diff
@@ -235,6 +235,17 @@ process_arg(Context& ctx,
     return nullopt;
   }
 
+  // Ignore Xcode -ivfsoverlay <path to yaml file> to not detect multiple input files
+  if (args[i] == "-ivfsoverlay") {
+    i++;
+    if (i == args.size()) {
+      log("-ivfsoverlay lacks an argument");
```

Comment:
Just tested this patch and found that probably this need to be `LOG` instead of `log`.

Reply:
Or probably `LOG_RAW`.

Reply:
Did you get a compile error or a different issue? (Can easily change this, just curious)

We have the current fork in production now and it greatly speeds up our compile times without any adverse effects. (I promised to report back - still want to give this another week or two to see if it really doesn't cause surprises).

I wrote about the process in https://pspdfkit.com/blog/2020/faster-compilation-with-ccache/

Reply:
Yes, I got an error with `cmake`, like `error: no matching function for call to 'log'`.
Also I rebased the patch on top of current tip branch, that may be related, and probably prefer to do anyways?